frontend/lib/src/components/widgets/DataFrame/styled-components.ts

```diff
@@ -43,5 +43,34 @@ export const StyledResizableContainer =
         ["overflowX" as any]: "auto !important",
         ["overflowY" as any]: "auto !important",
       },
+      "& .gdg-seveqep": {
+        // Make the search field more responsive to the grid width and use
+        // rem units for everything.
+        maxWidth: "19rem",
```

Comment:
Could you please leave a comment about how we landed at `19rem`? Even if it is just `We picked 19rem arbitrarily because it looked good on a laptop screen` or whatever. This will help others in the future in case this has to be revisited